test/parallel/test-runner-output.mjs

```diff
@@ -93,7 +93,7 @@ const tests = [
   { name: 'test-runner/output/abort_hooks.js' },
   { name: 'test-runner/output/describe_it.js' },
   { name: 'test-runner/output/describe_nested.js' },
-  { name: 'test-runner/output/eval_dot.js' },
+  { name: 'test-runner/output/eval_dot.js', transform: specTransform },
```

Comment:
Perhaps the name should now be changed as this is not only used by spec reporter tests